### ISSUE TITLE
Review-HAZ-Team[30]

### ISSUE THREAD

**Issue by ahmed-nazir** [edwin-do/capstoneTeam30#32]
The reviewer's capstone group TA is @tomashetMcMaster.
In the FMEA table I found the references a little hard to follow, some of the Hazards which are separate from each other go by H1-1,H1-2, etc. but the hazards which are part of the same component are labeled H2,H3,H4

Also for the Hazard for the sensors (Nanovoltmeter, temperature sensor) I think there could be a couple more potential hazards like interference/noise from external devices which would skew the results that you are getting from the sensors. I think those components can have other potentials hazards other then them just failing to read values.

**Comment by edwin-do** [comment]
Added in Final Revision - Awaiting merge before closing issue

**Comment by edwin-do** [comment]
Merged
